test/package.json

```diff
@@ -13,8 +13,10 @@
     "bedrock-docs": "^2.1.0",
     "bedrock-express": "^2.0.6",
     "bedrock-identity": "^4.6.0",
+    "bedrock-injector": "^1.0.0",
```

Comment:
We need this anymore?

Reply:
yes, that's the test package file, bedrock-injector is required by bedrock-ledger-node.